content/testsuite/testsuite.go

```diff
@@ -367,6 +367,7 @@ func checkLabels(ctx context.Context, t *testing.T, cs content.Store) {
 	labels := map[string]string{
 		"k1": "v1",
 		"k2": "v2",
+
```

Comment:
Opened a ticket upstream for the `gofmt` change; https://github.com/golang/go/issues/26228